<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AutoHeader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.icns</normaloff>logo.icns</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="forceSingleLine">
           <property name="text">
            <string>Single line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="safeModeCheckBox">
           <property name="text">
            <string>Safe mode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="selectButton">
           <property name="text">
            <string>Select files</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="templateComboBox">
           <property name="currentText">
            <string>--Select Template--</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
           </property>
           <item>
            <property name="text">
             <string>--Select Template--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Standard header</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Standard header (alternative)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>See LICENSE refrence</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Import header</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="fileListWidget">
           <property name="styleSheet">
            <string notr="true"> border-radius:10px; background-color: palette(base);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="fileEditor">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>689</width>
             <height>356</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"> border-radius:10px; background-color: palette(base);</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhMultiLine|Qt::ImhPreferLatin</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
